### ISSUE TITLE
Disable Map Dimming

### ISSUE THREAD

**Issue by karai17** [GoldpawsStuff/ClassicWorldMapEnhanced#10]
Is there any way to disable the map from dimming while moving? Really not liking that feature and the previous map add-on I used is no longer working in TBC.

**Comment by goldpaw** [comment]
Yeah, we more or less talked about it on our patron yesterday too. I'll add the option to disable the movement fading! :)

**Comment by karai17** [comment]
Thank you!
